dev_guide/pod_autoscaling.adoc

```diff
@@ -118,6 +119,64 @@ spec:
 <7> The percentage of the requested CPU that each pod should ideally be using
 ====
 
+[[pod-autoscaling-memory]]
+== Autoscaling for Memory Utilization
+
+Use the `oc autoscale` command and specify the maximum number of pods to run at
```

Comment:
might want to remove this sentence, considering the sentence below ("Unlike CPU-based autoscaling").